osf_tests/test_user.py

```diff
@@ -2205,16 +2209,26 @@ def test_validate_social_profile_website_many_different(self):
             self.user.social = {'profileWebsites': [should_fail]}
             try:
                 with pytest.raises(ValidationError):
-                    self.user.save()
+                    with mock.patch.object(spam_tasks.requests, 'head'):
+                        self.user.save()
             except AssertionError:
                 fails_at_end = True
                 print('\"' + should_fail + '\" passed but should have failed while testing that the validator ' + data['testsNegative'][should_fail])
         if fails_at_end:
             raise
 
+        # Not all domains that are permissable are possible to use as spam,
+        # some are correctly not extracted and not kept in notable domain so spot
+        # check some, not all, because not all `testsPositive` urls should be in
+        # NotableDomains
+        assert NotableDomain.objects.all().count() > previous_number_of_domains
```

Comment:
Minor: Is there a fixed number difference here? e.g. if we expect two to be added, it would be better to `assert NotableDomain.objects.count() == previous_number_of_domains + 2`